acquia_cms.profile

```diff
@@ -266,10 +297,19 @@ function acquia_cms_install_ui_kit(array $install_state) {
   }
 }
 
+/**
+ * Method that calls another method to capture the installation start time.
+ */
+function acquia_cms_install_ui_kit_finished($install_state) {
```

Comment:
@panshulK Why are we calling acquia_cms_install_ui_kit_finished() function two times:

1. `'finished' => 'acquia_cms_set_install_finished_time',`
2. `  $tasks['install_finished']['function'] = 'acquia_cms_install_ui_kit_finished';`

Reply:
@vishalkhode1 One is called for UI and another one is called when the CLI install runs.

Reply:
Ah ok. Make sense.